src/main/java/zd/zero/waifu/motivator/plugin/listeners/IdleEventListener.kt

```diff
@@ -60,9 +61,17 @@ class IdleEventListener(private val project: Project) : Runnable, Disposable {
                         project,
                         asset,
                         createAlertConfiguration()
-                    ).setListener {
-                        isEventDisplayed = false
-                    }.motivate()
+                    ).setListener(
+                        object : MotivationListener {
```

Comment:
This was an unrelated bug that I discovered by copy-pasting this class. When the notifications are turned off and the Idle event is triggered. The `isEventDisplayed` never gets set back to false, because no notifications where dismissed.